components/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt/src/test/java/org/wso2/carbon/identity/api/resource/mgt/dao/APIResourceManagementDAOImplTest.java

```diff
@@ -243,16 +351,32 @@ public void testIsScopeExistById(Integer tenantId, boolean expected) throws Exce
     public Object[][] deleteAPIResourceByIdData() {
 
         return new Object[][]{
-                {TENANT_ID, false}
+                {TENANT_ID, false, false},
+                {TENANT_ID, true, false}
```

Comment:
Is this test correct? in sub org level to we store API resource?

Reply:
No we dont. Here what the test is adding a resource and deleting that. The `isOrganization` property is for mocking purposes.